activesupport/test/core_ext/module_test.rb

```diff
@@ -417,7 +417,7 @@ def initialize(place)
     assert_not_respond_to place, :street
     assert_not_respond_to place, :city
 
-    assert place.respond_to?(:street, true) # Asking for private method
```

Comment:
This is the reason why we can not use `assert_respond_to`. Looks not a noise for me.

Reply:
Yup. Agree with @kamipo

Reply:
We use `respond_to?` for the same reason in 3 more tests(in `activesupport/test/core_ext/module_test.rb`). Do you think we should explain it in each of them?

Reply:
I agree, I'll revert this cosmetic changes and won't add additional ones then